docs/pages/en-us/Getting-Started.md

```diff
@@ -440,14 +228,14 @@ A generated client-side library should be referenced in the normal way in the ap
 	<dependency>
 	    <groupId>com.hpe.sampleapp</groupId>
 	    <artifactId>sampleapp-audit</artifactId>
-	    <version>1.0-SNAPSHOT</version>
+	    <version>1.0.0-SNAPSHOT</version>
 	</dependency>
 
 ### Audit Connection
 
 Regardless of whether you choose to use a generated client-side library, or to use `caf-audit` directly, you must first create an `AuditConnection` object.
 
-This object represents a logical connection to persistent storage (that is, Kafka in the current implementation). It is a thread-safe object. ***Please take into account that this object requires some time to construct. The application should hold on to it and re-use it, rather than constantly re-construct it.***
+This object represents a logical connection to persistent storage (that is, Elasticsearch in the current implementation). It is a thread-safe object. ***Please take into account that this object requires some time to construct. The application should hold on to it and re-use it, rather than constantly re-construct it.***
```

Comment:
This object represents a logical connection to persistent storage  
Update to:  
This object represents a logical connection to the datastore